examples/00-EDB/04_edb_parametrized_design.py

```diff
@@ -307,36 +283,28 @@
 )
 
 ###############################################################################
-# Set Differential Pairs.
-# ~~~~~~~~~~~~~~~~~~~~~~
-# Define the differential pairs to be used in the postprocessing.
+# Define the differential pairs to be used to calculte differential and common mode
+# s-parameters.
 h3d.set_differential_pair(diff_name="In", positive_terminal="wave_port_1:T1", negative_terminal="wave_port_1:T2")
 h3d.set_differential_pair(diff_name="Out", positive_terminal="wave_port_2:T1", negative_terminal="wave_port_2:T2")
 
 ###############################################################################
-# Start HFSS solver
-# ~~~~~~~~~~~~~~~~~
-# Start the HFSS solver by uncommenting the ``h3d.analyze()`` command.
+# Solve the project.
 
 h3d.analyze()
 
-
 ###############################################################################
-# Generate Plot
-# ~~~~~~~~~~~~~
-# Generate the plot of differential pairs.
+# Plot the results and shut down the Electronics Desktop.
 
 solutions = h3d.post.get_solution_data(["dB(S(In,In))", "dB(S(In,Out))"], context="Differential Pairs")
 solutions.plot()
-
-
-
-
-
 h3d.release_desktop()
 
 ###############################################################################
 # Note that the ground nets are only connected to each other due
 # to the wave ports. The problem with poor grounding can be seen in the
-# S-parameters. Try to modify this script to add ground vias and eliminate
-# the resonance.
+# S-parameters. This example can be downloaded as a Jupyter Notebook, so you can modify it. Try changing parameters or adding ground vias to improve performance.
```

Comment:
```suggestion
# S-parameters. This example can be downloaded as a Jupyter notebook so that you can modify it. Try changing parameters or adding ground vias to improve performance.
```